fauna/lib/PaginatedQuery.js

```diff
@@ -0,0 +1,30 @@
+import { query as q } from 'faunadb'
+
```

Comment:
One of the nice things about managing Fauna UDFs in code is you can create reusable lib functions like this one. Any query can achieve pagination without copy/pasting the yuck below.

Reply:
Woooow, thanks for this!